public/components/monitoring/model_deployment_table.tsx

```diff
@@ -138,6 +140,34 @@ export const ModelDeploymentTable = ({
           );
         },
       },
+      {
+        field: 'id',
+        name: 'Model ID',
+        width: '18%',
```

Comment:
Should we let width self-adaptive or specific?

Reply:
Do you mean self-adaptive by model id length? I think it's hard to do that. The percentage column width value will follow by the table width.  It will shrink if window size reduced. The model id will be truncated.
![image](https://github.com/opensearch-project/ml-commons-dashboards/assets/4034161/2b932c95-6057-49ae-be47-79851b356fc3)